charts/bitbucket-runner-autoscaler/templates/job-template.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bitbucket-runner-template
+  namespace: {{ .Values.namespace }}
+data:
+  job.yaml.template: |
+    apiVersion: v1
```

Comment:
Would it be beneficial to store the whole value in values.yaml and process it here with `tpl` function?